feat: saml application configuration for login version (#9351)
zitadel/zitadel: docs/docs/guides/integrate/login-ui/oidc-standard.mdx

The auth request includes all the relevant information for the OIDC standard and in this example we also have a login hint for the login name "minnie-mouse".

You now have to proxy the auth request from your own UI to the authorize Endpoint of ZITADEL.

> For more information have a look under [OIDC Proxy](./typescript-repo#oidc-proxy) for the necessary headers.

Reviewer: ```suggestion
For more information, see [OIDC Proxy](./typescript-repo#oidc-proxy) for the necessary headers.
```
